pkg/oci/utils.go

```diff
@@ -286,6 +287,35 @@ func (spec *CompatOCISpec) PodID() (string, error) {
 	return "", fmt.Errorf("Could not find pod ID")
 }
 
+func ociResourceUpdateRuntimeConfig(ocispec CompatOCISpec, config *RuntimeConfig) error {
+	if ocispec.Linux == nil {
+		return nil
+	}
+
+	if ocispec.Linux.Resources == nil {
+		return nil
+	}
+
+	if ocispec.Linux.Resources.Memory == nil {
+		return nil
+	}
+
+	if ocispec.Linux.Resources.Memory.Limit == nil {
+		return nil
+	}
+
+	memBytes := *ocispec.Linux.Resources.Memory.Limit
+
+	// round up memory bytes to MB
+	mem := int(math.Ceil(float64(memBytes) / 1024 / 1024))
```

Comment:
I'm ok with less precision, because this is trivial.

But @grahamwhaley suggested that we should round up at https://github.com/clearcontainers/runtime/pull/390#discussion_r132192014. I just quoted here:

> maybe we should round this up. Docker allows fine granularity (down to bytes). I think it is probably better we allocate more than requested rather than too little

@grahamwhaley what do you think now?